### DISCUSSION TITLE
Implement session management with Clerk on a non-JS environment

### DISCUSSION THREAD

**Discussion by ghost** [clerk/javascript#2960] [Support request - DO NOT OPEN HERE]
Hey everyone,

I was able to create an [OAuth app as described in this guide](https://clerk.com/docs/advanced-usage/clerk-idp) and register a Drupal instance as a client to it. It works great.

Now I would like to implement session management on Drupal (PHP) using Clerk but I can't find a guide describing how to use the API for it. Is it possible? Should I use the backend or the frontend API from Clerk docs? Is there a guide that could help me maybe?

Thank you

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
